spec/system/admin/tag_rules_spec.rb

```diff
@@ -187,7 +187,7 @@
         select2_select 'VISIBLE',
                        from: "enterprise_tag_rules_attributes_4_preferred_matched_shipping_methods_visibility"
       end
-
+=begin
```

Comment:
Was this meant to be left in?

Reply:
I'm not sure. I've opted to keep the `drag_to` related assertions commented out till we find a solution for issue https://github.com/openfoodfoundation/openfoodnetwork/issues/8463

But maybe this is missing a comment line to explain the context. What do you think?

Reply:
Ah I see! Okay, lets go with this for now :+1: